src/components/Types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export type Todo = {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User | undefined
```

Comment:
```suggestion
  user?: User
```
If some field is optional you can mark it with question mark